MediaAdminBundle/Controller/Api/MediaController.php

```diff
@@ -113,13 +113,22 @@ public function uploadAction($folderId, Request $request)
 
         if ($uploadedFile && $filename = $saveMediaManager->getFilenameFromChunks($uploadedFile)) {
 
-            $media = $this->container->get('open_orchestra_media.manager.media')
-                ->createMediaFromUploadedFile($uploadedFile, $filename, $folderId);
+            if ($saveMediaManager->isFileAllowed($filename)) {
+                $media = $this->container->get('open_orchestra_media.manager.media')
+                    ->createMediaFromUploadedFile($uploadedFile, $filename, $folderId);
+
+                return $this->get('open_orchestra_api.transformer_manager')
+                    ->get('media')->transform($media);
+            } else {
+                $translator = $this->container->get('translator');
 
-            return $this->get('open_orchestra_api.transformer_manager')
-                ->get('media')->transform($media);
+                return new Response(
```

Comment:
As the Api\Serialize annotation as been put on the full controller, you cannot return a Response, you should either : 

 - put the annotation on each controller action
 - return an empty array or a facade